src/jquery.fancytree.dnd.js

```diff
@@ -312,22 +316,22 @@ $.ui.fancytree.registerExtension({
 				.prependTo($(this.$div).parent());
 //                .prependTo("body");
 
-			if( glyph ) {
+			if( glyphOpt ) {
 				// instData.$dropMarker.addClass(glyph.map.dragHelper);
 				instData.$dropMarker
-					.addClass(glyph.map.dropMarker);
+					.addClass(glyphOpt.map.dropMarker);
 			}
 		}
 		if( hitMode === "after" || hitMode === "before" || hitMode === "over" ){
-			markerOffsetX = -24;
+			markerOffsetX = dndOpt.dropMarkerOffset || 0;
 			switch(hitMode){
 			case "before":
 				markerAt = "top";
-				markerOffsetX -= 16;
+				markerOffsetX += (dndOpt.dropMarkerOffsetBefore || 0);
 				break;
 			case "after":
 				markerAt = "bottom";
-				markerOffsetX -= 16;
+				markerOffsetX += (dndOpt.dropMarkerOffsetAfter || 0);
```

Comment:
This offset will always be the same for 'after' and 'before' so I suggest to combine it into one single option.
Also we could use two positive defaults:
dropMarkerOffsetX: 8
dropMarkerOffsetIndentX: 16

before/after offset will be dropMarkerOffsetX
otherwise dropMarkerOffsetX + dropMarkerOffsetIndentX